test/Microsoft.Framework.Internal.Test/BufferedHtmlContentTest.cs

```diff
@@ -59,38 +59,6 @@ public void AppendEncoded_DoesNotGetWrittenAsEncoded()
         }
 
         [Fact]
-        public void AppendLine_String_NewLineDoesNotGetEncoded()
```

Comment:
Since you didn't add `HtmlContentBuilderExtensions` tests in HttpAbstractions, I suggest keeping these here. At least we'd have some coverage of those methods.